core_mama.py

```diff
@@ -166,18 +166,17 @@ def run_mama_method(betas, omega, sigma):
     """
     Runs the core MAMA method to combine results and generate final, combined summary statistics
 
-    :param harm_sumstats: TODO(jonbjala)
-    :param omega: TODO(jonbjala)
-    :param sigma: TODO(jonbjala)
+    :param betas: MxP matrix of beta values (M = # of SNPs, P = # of ancestries)
+    :param omega: MxPxP matrix of omega values (M = # of SNPs, P = # of ancestries)
+    :param sigma: MxPxP matrix of sigma values (M = # of SNPs, P = # of ancestries)
 
     :return: Tuple containing:
                  1) Result ndarray of betas (MxP) where M = SNPs and P = populations
                  2) Result ndarray of beta standard errors (MxP) where M = SNPs and P = populations
     """
 
     # Get values for M and P (used to keep track of slices / indices / broadcasting)
-    M = omega.shape[0]
-    P = omega.shape[1]
+    M, P, *extra_dimensions = omega.shape
```

Comment:
Out of curiosity, when would omega have different dimensions?

Reply:
Well, right now it just has one more, and likely won't have any others, but if we ever did add more for whatever reason, this would be accurate / not misleading without any extra changes.  Right now we don't do anything with those extra dimensions anyway, so lumping them together doesn't do any harm at this point.